src/chrome/content/rules/Wdr.de.xml

```diff
@@ -59,23 +64,32 @@
 	<target host="deviceids-medt-id3.wdr.de" />
 	<target host="deviceids-medt-id4.wdr.de" />
 	<target host="deviceids-medt.wdr.de" />
+	<target host="digit.wdr.de" />
 	<target host="elefant.wdr.de" />
 	<target host="elefantenapp.wdr.de" />
 	<target host="exporte.wdr.de" />
+	<target host="kandidatencheck.wdr.de" />
+	<target host="kiraka.wdr.de" />
+	<target host="mobiltext.wdr.de" />
 	<target host="presse.wdr.de" />
-	<target host="rio.sportschau.de" />
+	<target host="reportage.wdr.de" />
+	<target host="superkuehe.wdr.de" />
+	
 	<target host="www.einslive.de" />
 	<target host="www.funkhauseuropa.de" />
-	<target host="www.sportschau.de" />
-	<target host="www.wdr.de" />
+	
+	<target host="planet-wissen.de" />
+	<target host="www.planet-wissen.de" />
+	
+	<target host="rio.sportschau.de" />
+	
 	<target host="www.wdr2.de" />
 	<target host="www.wdr3.de" />
 	<target host="www.wdr4.de" />
 	<target host="www.wdr5.de" />
-	<target host="www01.wdr.de" />
-	<target host="www1.sportschau.de" />
```

Comment:
Please keep this host.

Reply:
www01.wdr.de only moved to a other position.